curriculum/challenges/english/25-front-end-development/quiz-js-a11y/68376949899fcbad3c9cac23.md

```diff
@@ -17,438 +17,259 @@ To pass the quiz, you must correctly answer at least 18 of the 20 questions belo
 
-Placeholder answer
+`aria-expanded`
 
 ### --question--
 
 #### --text--
 
-Placeholder question
+Which two attributes can be used in combination with `aria-expanded` to establish a programmatic connection between the controlling element and the element it controls?
 
 #### --distractors--
 
-Placeholder distractor 1
+`aria-labelledBy` and `aria-owns`
 
 ---
 
-Placeholder distractor 2
+`aria-controls` and `aria-label`
 
 ---
 
-Placeholder distractor 3
+`aria-live` and `aria-owns`
 
 #### --answer--
 
-Placeholder answer
+`aria-controls` and `aria-owns`
 
 ### --question--
 
 #### --text--
 
-Placeholder question
+Which of the following attributes creates a live region on your page which can be used to notify screen reader users of dynamic content changes in the live region as they occur?
 
 #### --distractors--
 
-Placeholder distractor 1
+`aria-live-regions`
 
 ---
 
-Placeholder distractor 2
+`aria-lives`
 
 ---
 
-Placeholder distractor 3
+`aria-live-region`
 
 #### --answer--
 
-Placeholder answer
+`aria-live`
 
 ### --question--
 
 #### --text--
 
-Placeholder question
+Which of the following attributes is used to indicate that an element is disabled only to people using assistive technologies, such as screen readers?
 
 #### --distractors--
 
-Placeholder distractor 1
+```html
+<div role="button" tabindex="-1" aria-disables="true">Edit</div>
+```
 
 ---
 
-Placeholder distractor 2
+```html
+<div role="button" tabindex="-1" aria-disabling="true">Edit</div>
+```
 
 ---
 
-Placeholder distractor 3
+```html
+<div role="button" tabindex="-1" aria-disable="true">Edit</div>
+```
 
 #### --answer--
 
-Placeholder answer
+```html
+<div role="button" tabindex="-1" aria-disabled="true">Edit</div>
+```
 
 ### --question--
 
 #### --text--
 
-Placeholder question
+Which of the following is a valid use of the `aria-expanded` attribute?
 
 #### --distractors--
 
-Placeholder distractor 1
+```html
+<button aria-expanded="expand">Menu</button>
+```
 
 ---
 
-Placeholder distractor 2
+```html
+<button aria-expanded="allow">Menu</button>
+```
 
 ---
 
-Placeholder distractor 3
+```html
+<button aria-expanded="set">Menu</button>
+```
 
 #### --answer--
 
-Placeholder answer
+```html
+<button aria-expanded="true">Menu</button>
+```
 
 ### --question--
 
 #### --text--
 
-Placeholder question
+Which of the following attributes is used to create a programmatic relationship between an element that controls the presence of another element on the page?
 
 #### --distractors--
 
-Placeholder distractor 1
+`aria-set`
 
 ---
 
-Placeholder distractor 2
+`aria-region`
 
 ---
 
-Placeholder distractor 3
+`aria-expanded`
 
 #### --answer--
 
-Placeholder answer
+`aria-controls`
 
 ### --question--
 
 #### --text--
 
-Placeholder question
+Which of the following is NOT a common use case for live regions?
 
 #### --distractors--
 
-Placeholder distractor 1
+Messages that are displayed after an action has been taken(such as an error message or confirmation).
```

Comment:
```suggestion
Messages that are displayed after an action has been taken (such as an error message or confirmation).
```

I prefer spaces before parenthesis. I think that's standard - but I'm fine leaving them if you want.

Reply:
Looks like Dario already suggested these